bing_rewards/options.py

```diff
@@ -137,9 +137,10 @@ def parse_args() -> Namespace:
     )
     p.add_argument(
         '--profile',
-        help='Sets the chrome profile for launch',
+        help='Sets one or more chrome profiles to run sequentially (space separated)',
         type=str,
-        default='',
+        nargs='*',
```

Comment:
It might make sense to use nargs='+' here. That would make --profile without a argument invalid, wouldn't require a default empy list, and might simplify logic for the case when no --profile flag is set. 
https://docs.python.org/3/library/argparse.html#nargs